src/types/types.ts

```diff
@@ -201,9 +201,12 @@ export interface PageInfo {
   rowsOnCurrentPage: number;
 }
 
-interface FontStyling {
+export interface FontStyling {
   fontSize: string;
   fontFamily: string;
+  fontWeight: "600" | "normal";
+  fontStyle: "italic" | "normal";
+  textDecoration: "underline" | "none";
```

Comment:
maybe overkill but, you can have a basicFontStyling interface that this one extends and use it below instead of pick

Reply:
I'll wait and see how it plays out when I add all these styling options for some of the others sections as well